domain/application/service/application_test.go

```diff
@@ -577,15 +581,69 @@ func (s *applicationServiceSuite) TestAddUnits(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *applicationServiceSuite) TestGetUnitUUID(c *gc.C) {
```

Comment:
Could you please also add a small test with an error returned from state, just to check that we preserve it?

Reply:
Done